gk/fib.c

```diff
@@ -1864,15 +1954,33 @@ l_add_grantor_entry_lb(lua_State *l)
 	prefix_info.str = prefix;
 	prefix_info.len = parse_ip_prefix(prefix, &prefix_info.addr);
 
-	ret = add_fib_entry_numerical(&prefix_info,
-		gt_addrs, gw_addrs, tbl_size, GK_FWD_GRANTOR,
-		RTPROT_STATIC, gk_conf);
+	if (overwrite) {
+		ret = update_fib_entry_numerical(&prefix_info,
+			gt_addrs, gw_addrs, tbl_size, GK_FWD_GRANTOR,
+			RTPROT_STATIC, gk_conf);
+	} else {
+		ret = add_fib_entry_numerical(&prefix_info,
+			gt_addrs, gw_addrs, tbl_size, GK_FWD_GRANTOR,
+			RTPROT_STATIC, gk_conf);
+	}
 	if (ret < 0)
 		luaL_error(l, "Could not add FIB entry; check Gatekeeper log");
```

Comment:
Update message with `"Could not add or update FIB entry; check Gatekeeper log"`.